modules/aruco/samples/samples_utility.hpp

```diff
@@ -0,0 +1,110 @@
+// This file is part of OpenCV project.
+// It is subject to the license terms in the LICENSE file found in the top-level directory
+// of this distribution and at http://opencv.org/license.html.
+
+#ifndef _SAMPLES_UTILITY_HPP_
+#define _SAMPLES_UTILITY_HPP_
+
+#include <opencv2/highgui.hpp>
+#include <opencv2/aruco.hpp>
+#include <opencv2/calib3d.hpp>
+#include <ctime>
+
+inline bool readCameraParameters(std::string filename, cv::Mat &camMatrix, cv::Mat &distCoeffs) {
```

Comment:
inline -> static

Reply:
CI throw warning to static the functions:
`warning: defined but not used [-Wunused-function]`
what to do in this case?

Reply:
![image](https://user-images.githubusercontent.com/22337800/146204532-086ce98f-4dcb-4785-9791-4a357d5acdf7.png)

Reply:
Maybe I should use `static inline`?

Reply:
These attached messages comes from tests compilation, not sample code.

Anonymous namespace may help.

Reply:
Anonymous namespace didn't help ((( (warnings are still there due to tests)

Reply:
fixed
include of `aruco_samples_utility.hpp` is removed from tests